src/core/account/account-pixie.js

```diff
@@ -130,7 +130,8 @@ const accountPixie: TamePixie<AccountProps> = combinePixies({
         const changeLists = await Promise.all(
           accountWalletInfos.map(info => syncStorageWallet(ai, info.id))
         )
-        const changes: string[] = [].concat(...changeLists)
+        const changes: string[] = []
```

Comment:
Also why isn't this simpler:

```ts
const changes: string[] = [...changeLists]
```

Reply:
That is not the same result. We are concatenating a bunch of little arrays into one big array.